packages/css/README.md

```diff
@@ -4,17 +4,14 @@ The [@emotion/css](https://www.npmjs.com/package/@emotion/css) package is framew
 
 ## Table of Contents
 
-- [@emotion/css](#emotioncss)
-  - [Table of Contents](#table-of-contents)
-  - [Quick Start](#quick-start)
-  - [API](#api)
-    - [css](#css)
-      - [String Styles](#string-styles)
-      - [Object Styles](#object-styles)
-      - [Array of Object Styles](#array-of-object-styles)
-    - [Global Styles](#global-styles)
-    - [Animation Keyframes](#animation-keyframes)
-    - [cx](#cx)
+- [Quick Start](#quick-start)
+- [API](#api)
+  - [Generate Class Names — `css`](#css)
+  - [Global Styles — `injectGlobal`](#global-styles)
+  - [Animation Keyframes — `keyframes`](#animation-keyframes)
+  - [Composing Class Names — `cx`](#cx)
+- [Server Side Rendering](https://emotion.sh/docs/ssr#api)
```

Comment:
those 2 last things should point to a local repo paths, shouldnt they? at the moment they point to external site

Reply:
The site removes the `https://emotion.sh` so it points to the site on npm/GitHub and it points to local url on the site, e.g. https://deploy-preview-1675--emotion.netlify.com/docs/@emotion/css

Reply:
It won't work directly when using GitHub or exploring a checked out repository though.

Reply:
Yeah, that's true but the alternatives are:
- absolute path relative to the root or relative path from the README - works on GH and I think the site but is broken on npm
- GitHub url to the root - links to the file on master always so it's not on the right branch

So I think this solution is the best of the three